app/assets/javascripts/models/beta/task.js

```diff
@@ -0,0 +1,69 @@
+import httpService from '../../services/httpService';
+import changeCase from 'change-object-case';
+
+export function newTask(projectId, storyId, task) {
+  let newTask = {
```

Comment:
You can use `const` here instead of `let` since there is no reassignment. [See more about this](https://mathiasbynens.be/notes/es6-const#const-vs-let)